# Implementation of SpaceX {#ImplementSpaceX}

SpaceX function extimates shared and cluster specfic gene co-expression networks for spatial transcriptomics data. More details about the SpaceX method can be found in the main manuscript. See below for detailed discussion on installation of SpaceX package, Data inputs and outputs followed by an example.  


## Installation 
You can install the released version of SpaceX from (https://github.com/SatwikAch/SpaceX) with:
```{r, eval=FALSE}
devtools::install_github("SatwikAch/SpaceX")
library(SpaceX)
```

## Data inputs
Please make sure to provide both inputs as dataframe. 

The first input is **Gene_expression_mat** which is $N \times G$ dataframe. Here $N$ denotes the number of spatial locations and $G$ denotes number of genes. 

The second input is **Spatial_locations** is a dataframe which contains spatial coordinates.

The third input is **Cluster_annotations**.

## Output
You will obtain a list of objects as output.

**SigmaPhi** denotes the Shared Covariance matrix i.e. $\Sigma_{\Phi} = \Phi \Phi^{T}$.

**SigmaLambda** denotes the cluster specific Covaraince matrices i.e. $\Sigma_{\Psi^{c}} =  \Psi^{c} {\Psi^{c}}^{T}$.


## Example
Here we provide an example to run the SpaceX method. 
```{r, eval=FALSE}
# Reading the Breast cancer data

# Spatial locations
head(BC_loc)

# Gene expression for data
head(BC_count) 

# Data processing
G <-dim(BC_count)[2] # number of genes
N <-dim(BC_count)[1] # number of locations
```

Next, we'll apply the SpaceX method on the Breast cancer dataset.

```{r, eval=FALSE}
# Application to SpaceX method
BC_fit <- SpaceX(BC_count,BC_loc[,1:2],BC_loc[,3])

# SigmaPhi :: Shared Covariance matrix
# SigmaLambda :: Cluster specific Covaraince matrices
```
