<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A Methodology | Supplementary Materials for Gene Co-expression Network Estimation for Spatial Transcriptomics</title>
  <meta name="description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation for Spatial Transcriptomics." />
  <meta name="generator" content="bookdown 0.22.17 and GitBook 2.6.7" />

  <meta property="og:title" content="A Methodology | Supplementary Materials for Gene Co-expression Network Estimation for Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation for Spatial Transcriptomics." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A Methodology | Supplementary Materials for Gene Co-expression Network Estimation for Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation for Spatial Transcriptomics." />
  

<meta name="author" content="Satwik Acharyya, Xiang Zhou, Veera Baladandayuthapani" />


<meta name="date" content="2021-12-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="simulation.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Network Estimation in Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="appendix"><span><b>Supplementary Materials</b></span></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="A" data-path="method.html"><a href="method.html"><i class="fa fa-check"></i><b>A</b> Methodology</a>
<ul>
<li class="chapter" data-level="A.1" data-path="method.html"><a href="method.html#poisson-mixed-model"><i class="fa fa-check"></i><b>A.1</b> Poisson Mixed Model</a></li>
<li class="chapter" data-level="A.2" data-path="method.html"><a href="method.html#multi-study-factor-model-msfa"><i class="fa fa-check"></i><b>A.2</b> Multi-Study Factor Model (MSFA)</a></li>
<li class="chapter" data-level="A.3" data-path="method.html"><a href="method.html#multiplicative-gamma-shrinkage-prior"><i class="fa fa-check"></i><b>A.3</b> Multiplicative gamma shrinkage prior</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="simulation.html"><a href="simulation.html"><i class="fa fa-check"></i><b>B</b> Simulation Study</a>
<ul>
<li class="chapter" data-level="B.1" data-path="simulation.html"><a href="simulation.html#induced-correlation-study"><i class="fa fa-check"></i><b>B.1</b> Induced Correlation Study</a></li>
<li class="chapter" data-level="B.2" data-path="simulation.html"><a href="simulation.html#comparative-analysis-with-different-norm-measures"><i class="fa fa-check"></i><b>B.2</b> Comparative analysis with different norm measures</a></li>
<li class="chapter" data-level="B.3" data-path="simulation.html"><a href="simulation.html#estimation-of-latent-factors"><i class="fa fa-check"></i><b>B.3</b> Estimation of latent factors</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="realdata.html"><a href="realdata.html"><i class="fa fa-check"></i><b>C</b> Real Data Analysis</a>
<ul>
<li class="chapter" data-level="C.1" data-path="realdata.html"><a href="realdata.html#exploratory"><i class="fa fa-check"></i><b>C.1</b> Exploratory analysis of the datasets</a>
<ul>
<li class="chapter" data-level="C.1.1" data-path="realdata.html"><a href="realdata.html#merfish-data"><i class="fa fa-check"></i><b>C.1.1</b> Merfish Data</a></li>
<li class="chapter" data-level="C.1.2" data-path="realdata.html"><a href="realdata.html#breast-cancer-data"><i class="fa fa-check"></i><b>C.1.2</b> Breast Cancer Data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="realdata.html"><a href="realdata.html#communitydetection"><i class="fa fa-check"></i><b>C.2</b> Community detection</a></li>
<li class="chapter" data-level="C.3" data-path="realdata.html"><a href="realdata.html#list-of-hub-genes-and-edges"><i class="fa fa-check"></i><b>C.3</b> List of hub genes and edges</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="ImplementSpaceX.html"><a href="ImplementSpaceX.html"><i class="fa fa-check"></i><b>D</b> Implementation of SpaceX</a>
<ul>
<li class="chapter" data-level="D.1" data-path="ImplementSpaceX.html"><a href="ImplementSpaceX.html#installation"><i class="fa fa-check"></i><b>D.1</b> Installation</a></li>
<li class="chapter" data-level="D.2" data-path="ImplementSpaceX.html"><a href="ImplementSpaceX.html#data-inputs"><i class="fa fa-check"></i><b>D.2</b> Data inputs</a></li>
<li class="chapter" data-level="D.3" data-path="ImplementSpaceX.html"><a href="ImplementSpaceX.html#output"><i class="fa fa-check"></i><b>D.3</b> Output</a></li>
<li class="chapter" data-level="D.4" data-path="ImplementSpaceX.html"><a href="ImplementSpaceX.html#example"><i class="fa fa-check"></i><b>D.4</b> Example</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary Materials for Gene Co-expression Network Estimation for Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="method" class="section level1" number="1">
<h1><span class="header-section-number">A</span> Methodology</h1>
<p>In this section, we provide a detailed discussion of the estimation procedure for the SpaceX model equation 1 in section 2.1 of the paper. A full-scale MCMC will be computationally expensive on a complex hierarchical model. For computational advantage, we decompose the model into two parts (I) sPMM: spatial Poisson mixed model <span class="citation">(<a href="#ref-sun2017differential" role="doc-biblioref">Sun et al. 2017</a>)</span> and (II) MSFA: Multi-study factor analysis model <span class="citation">(<a href="#ref-de2018bayesian" role="doc-biblioref">De Vito et al. 2018</a>)</span>. We enable this model decomposition through a standard Gaussian random variable.</p>
<div id="poisson-mixed-model" class="section level2" number="1.1">
<h2><span class="header-section-number">A.1</span> Poisson Mixed Model</h2>
<p>We can break the SpaceX model and write the spatial Poisson mixed model as</p>
<p><span class="math display" id="eq:PMM-supp">\[\begin{align}
\begin{split}
&amp; \text{log}({\boldsymbol \lambda}^{c}_{g}) = {{\bf X}^{c}}^{T} {\boldsymbol \beta}^{c}_{g} + {\bf s}^{c} + {\bf z}^{c}_{g}, \\
&amp; {\boldsymbol \lambda}^{c}_{g} = ( \lambda^{c}_{1g}, \dots , \lambda^{c}_{N_{c}g} )^{T}, \\
&amp;{\bf s}^{c} = ( s^{c}_{1}, \dots , s^{c}_{N_{c}} )^{T} \sim \text{MVN}(0, \sigma_{1}^{2}\Omega^{c}(s)), \\
&amp;{\bf z}^{c}_{g} = (z^{1}_{1g}, \dots , z^{c}_{N_{c}g})^{T} \sim \text{MVN}(0, \sigma_{2}^{2} I_{N_{c} \times N_{c} }).
\end{split}
\tag{A.1}
\end{align}\]</span></p>
<p>Here <span class="math inline">\(\Omega^{c}(s_{1}, s_{2}) = \text{exp} (- \mid \mid s_{1} - s_{2} \mid \mid^{2} / 2 \rho^{2}_{c} ),\)</span> <span class="math inline">\(c = 1, \dots, C\)</span>. We estimate the length scale parameter of spatial kernel <span class="math inline">\(\rho_{c}\)</span> based on the steps discussed in section 1 of supplementary information in <span class="citation"><a href="#ref-sun2017differential" role="doc-biblioref">Sun et al.</a> (<a href="#ref-sun2017differential" role="doc-biblioref">2017</a>)</span>. Here <span class="math inline">\(Z^{c}_{g}\)</span> captures the cluster specific latent gene expressions and a multi-variate hierarchical modeling of <span class="math inline">\(Z^{c}_{g}(s_{i})\)</span> will help us to identify the gene co-expression network.</p>
</div>
<div id="multi-study-factor-model-msfa" class="section level2" number="1.2">
<h2><span class="header-section-number">A.2</span> Multi-Study Factor Model (MSFA)</h2>
<p>The 2nd stage of the modeling framework is multi-study factor analysis <span class="citation">(<a href="#ref-de2018bayesian" role="doc-biblioref">De Vito et al. 2018</a>)</span> which is provided as follows
<span class="math display" id="eq:MSFA-supp">\[\begin{align}
\begin{split}
&amp; \hat{\bf z}^{c}_{i} = \boldsymbol \Phi {\bf f}_{i} + \boldsymbol \Psi^{c} {\bf d}^{c}_{i} + {\bf e}^{c}_{i}, \\
&amp; {\bf f}_{i} \sim N_{K}(0, {\bf I}_{K}), \hspace{0.5cm} {\bf d}^{c}_{i} \sim N_{K_{c}}(0,{\bf I}_{K_{c}}),\\
&amp; {\bf e}^{c}_{i} \sim N_{G}(0,\boldsymbol \Xi_{c}), \hspace{0.5cm} \boldsymbol \Xi_{c} = \text{diag}(\xi^{c}_{1}, \dots, \xi^{c}_{G} ).
\end{split}
\tag{A.2}
\end{align}\]</span>
The marginal distribution of <span class="math inline">\(\hat{\bf z}^{c}_{i}\)</span> is a multivariate normal distribution with mean <span class="math inline">\(0\)</span> and covariance matrix <span class="math inline">\(\Sigma_{c}\)</span> s.t.
<span class="math display" id="eq:sigma-decomposition">\[\begin{align}
\Sigma_{c} = \Phi \Phi^{T} + \Psi^{c} {\Psi^{c}}^{T} + \Xi_{c} = \Sigma_{\Phi} + \Sigma_{\Psi^{c}} + \Xi_{c}
\tag{A.3}
\end{align}\]</span>
<span class="math inline">\(\Sigma_{\Phi} = \Phi \Phi^{T}\)</span> and <span class="math inline">\(\Sigma_{\Psi^{c}} = \Psi^{c} {\Psi^{c}}^{T}\)</span> are covariance of shared and cluster specific factors respectively. The decomposition of <span class="math inline">\(\Sigma_{c}\)</span> in <a href="method.html#eq:sigma-decomposition">(A.3)</a> is not a unique since we can set <span class="math inline">\(\Phi^{*} = \Phi Q\)</span> and <span class="math inline">\({\Psi^{*}}^{c} = \Psi^{c} Q_{c}\)</span> where <span class="math inline">\(Q\)</span> and <span class="math inline">\(Q_{c}\)</span> are square orthonormal matrices. This will also lead to the same decomposition <span class="math inline">\(\Sigma^{c} = \Phi^{*} {\Phi^{*}}^{T} + {\Psi^{*}}^{c} {{\Psi^{*}}^{c}}^{T} = \Phi \Phi^{T} + \Psi_{c} \Psi_{c}^{T}\)</span>. To overcome the indeterminacy through orthonormal matrices, the factor loading matrices are restricted to be lower triangular matrices <span class="citation">(<a href="#ref-geweke1996measuring" role="doc-biblioref">Geweke and Zhou 1996</a>; <a href="#ref-lopes2004bayesian" role="doc-biblioref">Lopes and West 2004</a>)</span>.</p>
</div>
<div id="multiplicative-gamma-shrinkage-prior" class="section level2" number="1.3">
<h2><span class="header-section-number">A.3</span> Multiplicative gamma shrinkage prior</h2>
<p>We follow the same steps from <span class="citation"><a href="#ref-de2018bayesian" role="doc-biblioref">De Vito et al.</a> (<a href="#ref-de2018bayesian" role="doc-biblioref">2018</a>)</span> and place multiplicative gamma shrinkage prior <span class="citation">(<a href="#ref-bhattacharya2011sparse" role="doc-biblioref">Bhattacharya and Dunson 2011</a>)</span> prior on the shared and cluster specific loading matrices i.e. <span class="math inline">\(\Phi\)</span> and <span class="math inline">\(\Psi_{c}\)</span> <span class="math inline">\(c=1,\dots,C\)</span>. The shared and cluster specific latent factors (<span class="math inline">\(K\)</span> and <span class="math inline">\(K_{c}\)</span> respectively) are selected following methodology described in section 3.3 of <span class="citation"><a href="#ref-de2018bayesian" role="doc-biblioref">De Vito et al.</a> (<a href="#ref-de2018bayesian" role="doc-biblioref">2018</a>)</span>. The multiplicative gamma prior on elements of shared covariance matrices are provided as follows
<span class="math display">\[\begin{equation}
\begin{split}
&amp; \phi_{gk} \mid \delta_{gk}, \eta_{k} \sim N(0, \delta_{gk}^{-1} \eta_{k}^{-1}), \hspace{0.5cm} g = 1, \dots , G, \hspace{0.25cm} k = 1, \dots, \infty, \\
&amp; \delta_{gk} \sim \Gamma(\frac{\nu}{2}, \frac{\nu}{2}) \hspace{0.5cm}  \eta_{k} = \prod_{j=1}^{k} \zeta_{j} \hspace{0.5cm} \zeta_{1} \sim \Gamma(a_{1},1) \hspace{0.5cm}  \zeta_{j} \sim \Gamma(a_{2},1), \hspace{0.2cm} j \ge 2. 
\end{split}
\end{equation}\]</span>
Here <span class="math inline">\(\delta_{gk}\)</span> is the local shrinkage parameter for <span class="math inline">\(G\)</span> column elements of <span class="math inline">\(k\)</span>th column and <span class="math inline">\(\eta_{k}\)</span> is the global shrinkage parameter where <span class="math inline">\(\zeta_{j}\)</span> <span class="math inline">\((j=1,2. \dots)\)</span> are independent. We repeat the same process to posit prior on the elements of cluster-specific loading matrices</p>
<p><span class="math display">\[\begin{equation}
\begin{split}
&amp; \psi^{c}_{gk_{c}} \mid \delta^{c}_{gk_{c}}, \eta^{c}_{k_{c}} \sim N(0, {\delta^{c^{-1}}_{gk_{c}}} {\eta^{c^{-1}}_{k_{c}}}), \hspace{0.5cm} g = 1, \dots , G, \hspace{0.2cm} k_{c} = 1, \dots, \infty \hspace{0.2cm} \text{and} \hspace{0.2cm} c = 1, \dots , C,\\
&amp; \delta^{c}_{gk_{c}} \sim \Gamma(\frac{\nu^{c}}{2}, \frac{\nu^{c}}{2}) \hspace{0.5cm}  \eta^{c}_{k_{c}} = \prod_{j=1}^{k_{c}} \zeta^{c}_{j} \hspace{0.5cm} \zeta^{c}_{1} \sim \Gamma(a^{c}_{1},1) \hspace{0.5cm}  \zeta^{c}_{j} \sim \Gamma(a^{c}_{2},1), \hspace{0.2cm} j \ge 2.
\end{split}
\end{equation}\]</span>
Here <span class="math inline">\(\delta^{c}_{gk_{c}}\)</span> <span class="math inline">\(\eta^{c}_{k_{c}}\)</span> are local and global parameters respectively and <span class="math inline">\(\zeta_{j}^{c}\)</span> <span class="math inline">\((c=1,2, \dots C)\)</span> are independent of each other. We determine <span class="math inline">\(K\)</span> and <span class="math inline">\(K_{c}\)</span> following methodology described in section 3.3 of <span class="citation"><a href="#ref-de2018bayesian" role="doc-biblioref">De Vito et al.</a> (<a href="#ref-de2018bayesian" role="doc-biblioref">2018</a>)</span>.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-bhattacharya2011sparse" class="csl-entry">
Bhattacharya, Anirban, and David B Dunson. 2011. <span>“Sparse Bayesian Infinite Factor Models.”</span> <em>Biometrika</em>, 291–306.
</div>
<div id="ref-de2018bayesian" class="csl-entry">
De Vito, Roberta, Ruggero Bellio, Lorenzo Trippa, and Giovanni Parmigiani. 2018. <span>“Bayesian Multi-Study Factor Analysis for High-Throughput Biological Data.”</span> <em>arXiv Preprint arXiv:1806.09896</em>.
</div>
<div id="ref-geweke1996measuring" class="csl-entry">
Geweke, John, and Guofu Zhou. 1996. <span>“Measuring the Pricing Error of the Arbitrage Pricing Theory.”</span> <em>The Review of Financial Studies</em> 9 (2): 557–87.
</div>
<div id="ref-lopes2004bayesian" class="csl-entry">
Lopes, Hedibert Freitas, and Mike West. 2004. <span>“Bayesian Model Assessment in Factor Analysis.”</span> <em>Statistica Sinica</em>, 41–67.
</div>
<div id="ref-sun2017differential" class="csl-entry">
Sun, Shiquan, Michelle Hood, Laura Scott, Qinke Peng, Sayan Mukherjee, Jenny Tung, and Xiang Zhou. 2017. <span>“Differential Expression Analysis for RNAseq Using Poisson Mixed Models.”</span> <em>Nucleic Acids Research</em> 45 (11): e106–6.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="simulation.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/01-Method-details.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
