<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>C Real Data Analysis | Supplementary Materials for Gene Co-expression Network Estimation in Spatial Transcriptomics</title>
  <meta name="description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation in Spatial Transcriptomics." />
  <meta name="generator" content="bookdown 0.22.17 and GitBook 2.6.7" />

  <meta property="og:title" content="C Real Data Analysis | Supplementary Materials for Gene Co-expression Network Estimation in Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation in Spatial Transcriptomics." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="C Real Data Analysis | Supplementary Materials for Gene Co-expression Network Estimation in Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="This containes all the supplementary materials for the paper named SpaceX: Gene Co-expression Network Estimation in Spatial Transcriptomics." />
  

<meta name="author" content="Satwik Acharyya, Xiang Zhou, Veera Baladandayuthapani" />


<meta name="date" content="2021-10-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="simulation-study.html"/>
<link rel="next" href="references.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Network Estimation in Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="appendix"><span><b>Supplementary Materials</b></span></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="A" data-path="methodology.html"><a href="methodology.html"><i class="fa fa-check"></i><b>A</b> Methodology</a>
<ul>
<li class="chapter" data-level="A.1" data-path="methodology.html"><a href="methodology.html#poisson-mixed-model"><i class="fa fa-check"></i><b>A.1</b> Poisson Mixed Model</a></li>
<li class="chapter" data-level="A.2" data-path="methodology.html"><a href="methodology.html#multi-study-factor-model-msfa"><i class="fa fa-check"></i><b>A.2</b> Multi-Study Factor Model (MSFA)</a></li>
<li class="chapter" data-level="A.3" data-path="methodology.html"><a href="methodology.html#multiplicative-gamma-shrinkage-prior"><i class="fa fa-check"></i><b>A.3</b> Multiplicative gamma shrinkage prior</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="simulation-study.html"><a href="simulation-study.html"><i class="fa fa-check"></i><b>B</b> Simulation Study</a>
<ul>
<li class="chapter" data-level="B.1" data-path="simulation-study.html"><a href="simulation-study.html#induced-correlation-study"><i class="fa fa-check"></i><b>B.1</b> Induced Correlation Study</a></li>
<li class="chapter" data-level="B.2" data-path="simulation-study.html"><a href="simulation-study.html#comparative-analysis-with-different-norm-measures"><i class="fa fa-check"></i><b>B.2</b> Comparative analysis with different norm measures</a></li>
<li class="chapter" data-level="B.3" data-path="simulation-study.html"><a href="simulation-study.html#estimation-of-latent-factors"><i class="fa fa-check"></i><b>B.3</b> Estimation of latent factors</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="real-data-analysis.html"><a href="real-data-analysis.html"><i class="fa fa-check"></i><b>C</b> Real Data Analysis</a>
<ul>
<li class="chapter" data-level="C.1" data-path="real-data-analysis.html"><a href="real-data-analysis.html#exploratory"><i class="fa fa-check"></i><b>C.1</b> Exploratory analysis of the datasets</a>
<ul>
<li class="chapter" data-level="C.1.1" data-path="real-data-analysis.html"><a href="real-data-analysis.html#merfish-data"><i class="fa fa-check"></i><b>C.1.1</b> Merfish Data</a></li>
<li class="chapter" data-level="C.1.2" data-path="real-data-analysis.html"><a href="real-data-analysis.html#breast-cancer-data"><i class="fa fa-check"></i><b>C.1.2</b> Breast Cancer Data</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="real-data-analysis.html"><a href="real-data-analysis.html#communitydetection"><i class="fa fa-check"></i><b>C.2</b> Community detection</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary Materials for Gene Co-expression Network Estimation in Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="real-data-analysis" class="section level1" number="3">
<h1><span class="header-section-number">C</span> Real Data Analysis</h1>
<p>We applied the SpaceX method on two spatial transcriptomics datasets which are obtained from the preoptic region of the mouse hypothalamus <span class="citation">(<a href="#ref-Moffitteaau5324" role="doc-biblioref">Moffitt et al. 2018</a>)</span> and the human breast cancer dataset <span class="citation">(<a href="#ref-staahl2016visualization" role="doc-biblioref">St√•hl et al. 2016</a>)</span>. Here we provide details of preprocessing and exploratory analysis of both datasets in section <a href="real-data-analysis.html#exploratory">C.1</a>. We illustrate the detailed application of the community detection algorithm on those two datasets in section <a href="real-data-analysis.html#communitydetection">C.2</a>.</p>
<div id="exploratory" class="section level2" number="3.1">
<h2><span class="header-section-number">C.1</span> Exploratory analysis of the datasets</h2>
<div id="merfish-data" class="section level3" number="3.1.1">
<h3><span class="header-section-number">C.1.1</span> Merfish Data</h3>
<p>The MERFISH dataset is obtained from the preoptic area of the mouse hypothalamus <span class="citation">(<a href="#ref-Moffitteaau5324" role="doc-biblioref">Moffitt et al. 2018</a>)</span>. The dataset consists of <span class="math inline">\(160\)</span> genes and corresponding gene expressions are measured in <span class="math inline">\(4975\)</span> spatial locations. There are <span class="math inline">\(7\)</span> pre-determined spatial clusters in the dataset named Astrocyte, Endothelial, Ependymal, Excitatory, Inhibitory, Immature, Mature, and the corresponding sizes are <span class="math inline">\(724\)</span>, <span class="math inline">\(503\)</span>, <span class="math inline">\(314\)</span>, <span class="math inline">\(1024\)</span>, <span class="math inline">\(1694\)</span>, <span class="math inline">\(168\)</span>, <span class="math inline">\(385\)</span> respectively. The dataset consists of <span class="math inline">\(2\)</span> more clusters named Microglia, Pericytes with cluster sizes <span class="math inline">\(90\)</span>, <span class="math inline">\(73\)</span> respectively which are less than <span class="math inline">\(100\)</span>. Those two clusters are removed from the dataset. After removing those two clusters, we have gene expressions from <span class="math inline">\(4812\)</span> locations corresponding to <span class="math inline">\(160\)</span> genes. There are no genes with more than <span class="math inline">\(95\%\)</span> zeros reads. Figure <a href="real-data-analysis.html#fig:zeroperMF">C.1</a> shows the violin plot of the percentage of zero reads among the genes for each cluster in the MERFISH dataset.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:zeroperMF"></span>
<img src="images/Zero_percentage_violin.jpeg" alt="Violin plot of percentage of zero reads among the genes for each cluster w.r.t. Merfish data." width="80%" />
<p class="caption">
Figure C.1: Violin plot of percentage of zero reads among the genes for each cluster w.r.t. Merfish data.
</p>
</div>
</div>
<div id="breast-cancer-data" class="section level3" number="3.1.2">
<h3><span class="header-section-number">C.1.2</span> Breast Cancer Data</h3>
<p>The human breast cancer dataset contains expression levels from <span class="math inline">\(5262\)</span> genes measured at <span class="math inline">\(250\)</span> locations <span class="citation">(<a href="#ref-staahl2016visualization" role="doc-biblioref">St√•hl et al. 2016</a>)</span>. We use the SPARK method with <span class="math inline">\(5\%\)</span> FDR cut-off on p-values to detect <span class="math inline">\(290\)</span> spatially expressed genes to carry forward our analysis.</p>
</div>
</div>
<div id="communitydetection" class="section level2" number="3.2">
<h2><span class="header-section-number">C.2</span> Community detection</h2>
<p>The community detection is a downstream analysis of the shared and cluster-specific networks which are obtained from the SpaceX method. The communities are detected by optimizing modularity over partitions in a network structure <span class="citation">(<a href="#ref-brandes2007modularity" role="doc-biblioref">Brandes et al. 2007</a>)</span>. Figure <a href="real-data-analysis.html#fig:comMERFISH">C.2</a> and <a href="real-data-analysis.html#fig:comBC">C.3</a> show the detected community modules from shared and cluster-specific co-expression networks for MERFISH and breast cancer data respectively.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:comMERFISH"></span>
<img src="images/merfishCom.png" alt="Shared and cell-type specific community detection for Merfish data" width="95%" />
<p class="caption">
Figure C.2: Shared and cell-type specific community detection for Merfish data
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:comBC"></span>
<img src="images/BC_Com.png" alt="Shared and cell-type specific community detection for Breast cancer data" width="80%" />
<p class="caption">
Figure C.3: Shared and cell-type specific community detection for Breast cancer data
</p>
</div>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-brandes2007modularity" class="csl-entry">
Brandes, Ulrik, Daniel Delling, Marco Gaertler, Robert Gorke, Martin Hoefer, Zoran Nikoloski, and Dorothea Wagner. 2007. <span>‚ÄúOn Modularity Clustering.‚Äù</span> <em>IEEE Transactions on Knowledge and Data Engineering</em> 20 (2): 172‚Äì88.
</div>
<div id="ref-Moffitteaau5324" class="csl-entry">
Moffitt, Jeffrey R., Dhananjay Bambah-Mukku, Stephen W. Eichhorn, Eric Vaughn, Karthik Shekhar, Julio D. Perez, Nimrod D. Rubinstein, et al. 2018. <span>‚ÄúMolecular, Spatial, and Functional Single-Cell Profiling of the Hypothalamic Preoptic Region.‚Äù</span> <em>Science</em> 362 (6416). <a href="https://doi.org/10.1126/science.aau5324">https://doi.org/10.1126/science.aau5324</a>.
</div>
<div id="ref-staahl2016visualization" class="csl-entry">
St√•hl, Patrik L, Fredrik Salm√©n, Sanja Vickovic, Anna Lundmark, Jos√© Fern√°ndez Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. <span>‚ÄúVisualization and Analysis of Gene Expression in Tissue Sections by Spatial Transcriptomics.‚Äù</span> <em>Science</em> 353 (6294): 78‚Äì82.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="simulation-study.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/03-Real-data-analysis.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
